py/torch_tensorrt/_compile.py

```diff
@@ -20,6 +20,8 @@
 
 DYNAMO_ENABLED = version.parse(sanitized_torch_version()) >= version.parse("2.1.dev")
 
+SAFE_MODE = torch.ops.tensorrt.get_safe_mode()
```

Comment:
Need to check for `torch_tensorrt._C` import before setting this. `_PY_RT_MULTI_DEVICE_SAFE_MODE` --> synced with the C++ flag through the functions.